src/tcgwars/logic/impl/gen4/GreatEncounters.groovy

```diff
@@ -1393,17 +1400,16 @@ public enum GreatEncounters implements LogicCardInfo {
           move "Psychic Balance", {
             text "If you have less cards in your hand than your opponent, draw cards until you have the same number of cards as your opponent. (If you have more or the same number of cards in your hand as your opponent, this attack does nothing.)"
             energyCost C
-            attackRequirement {}
             onAttack {
-              damage 0
```

Comment:
I haven't tested it yet but I wonder what the engine does when you do `draw -1`

Reply:
How so?

Reply:
The case when your hand size is larger than your opponent's hand size

Reply:
It's being used like this elsewhere on Holon Scientist and Lillie so I think it's okay